CommonMark.Console/Program.cs

```diff
@@ -61,6 +61,19 @@ static int Main(string[] args)
                     {
                         settings.OutputFormat = OutputFormat.SyntaxTree;
                     }
+                    else if (string.Equals(args[i], "--fat", StringComparison.OrdinalIgnoreCase))
```

Comment:
perhaps it would be simpler to automatically imply --fat when --extended is specified? Since the only reason why someone would want to use --fat is to use the placeholders that require the fat formatter.